locales/en-US/experiments.ftl

```diff
@@ -61,7 +61,7 @@ containersContributors7Title = Firefox UX
 containersContributors8Title = Firefox UX
 containersContributors9Title = Firefox UX
 firefoxlockboxDescription = Take your passwords everywhere with the Firefox Lockbox App for iPhone.
-firefoxlockboxIntroduction = <p>Take your passwords everywhere with Firefox Lockbox. This Test Pilot experiment gives you access to all the logins you've saved to Firefox, in a secure app on your iPhone. Just log in to Lockbox with your Firefox Account, and your saved usernames and passwords will sync to the app using 256-bit encryption. So you’ll have easy access to online accounts, even on the go.</p> <p><strong>Firefox Lockbox requires a Firefox Account in order to access your previously <a href="chrome://passwordmgr/content/passwordManager.xul">saved logins</a>. If you do not have an account set up, get started <a href="https://lockbox.firefox.com/faq.html#how-do-i-get-my-firefox-saved-logins-into-firefox-lockbox">here</a>.</strong></p> <p><em>Want to be notified when our Android app launches? Sign up <a href="https://goo.gl/forms/ZwLIfHSGLrYcM6k83">here</a>.</em></p>
+firefoxlockboxIntroduction = <p>Take your passwords everywhere with Firefox Lockbox. This Test Pilot experiment gives you access to all the logins you've saved to Firefox, in a secure app on your iPhone. Just log in to Lockbox with your Firefox Account, and your saved usernames and passwords will sync to the app using 256-bit encryption. So you’ll have easy access to online accounts, even on the go.</p> <p><strong>Firefox Lockbox requires a Firefox Account in order to access your previously <a href="chrome://passwordmgr/content/passwordManager.xul">saved logins</a>. If you do not have an account set up, get started <a href="https://lockbox.firefox.com/faq.html#how-do-i-get-my-firefox-saved-logins-into-firefox-lockbox">here</a>.</strong></p> <p><em>Want to be notified when our Android app launches? Sign up <a href="https://goo.gl/forms/ZwLIfHSGLrYcM6k83">here</a>.</em></p> <p>Please note, Firefox Lockbox is currently an English only app available in a few regions including Australia, New Zealand, United Kingdom, United States, and Canada.</p>
```

Comment:
If, on the other hand, the goal is to add this text only to English, I guess it makes sense. Note that new translation will include this text, existing ones won't know it exists…

Reply:
Yeah, if we revise an existing string, [we need to use the `<fieldname>_l10nsuffix` property in YAML to generate a new ID](https://github.com/mozilla/testpilot/blob/master/docs/content/reference.md#localization). That also invalidates the whole string for existing localizations too - because the ID changes - so we lose whatever we might have just merged in.